language/move-analyzer/src/symbols.rs

```diff
@@ -218,6 +222,8 @@ pub struct Symbols {
     file_use_defs: BTreeMap<PathBuf, UseDefMap>,
     /// A mapping from file hashes to file names
     file_name_mapping: BTreeMap<FileHash, Symbol>,
+    /// A mapping from filePath to function definitions
+    file_functions: BTreeMap<PathBuf, FunctionIdentTypeMap>,
```

Comment:
There is a problem here that needs addressing. We can have more than one module per file and each module may define a function with the same name. This essentially maps file paths to function names which is going to to be a problem if more than one function with the same name is defined in the same file.